package.json

```diff
@@ -53,7 +53,8 @@
     "ssb-replicate": "^1.3.3",
     "standard": "^17.1.0",
     "tap-arc": "^0.4.0",
-    "tape": "^4.16.2"
+    "tape": "^4.16.2",
+    "why-is-node-running": "^2.2.2"
```

Comment:
rm?

Reply:
was expecting to use it a lot more so didn't uninstall but sure i'll do it